logappender/README.md

```diff
@@ -0,0 +1,64 @@
+# RocketMQ-LogAppender   [![Build Status](https://travis-ci.org/rocketmq/rocketmq-logappender.svg?branch=master)](https://travis-ci.org/rocketmq/rocketmq-logappender) [![Coverage Status](https://coveralls.io/repos/github/rocketmq/rocketmq-logappender/badge.svg?branch=master)](https://coveralls.io/github/rocketmq/rocketmq-logappender?branch=master)
```

Comment:
Could we remove it if  we want it into parent modue, we can reuse rocektmq existing infra.